src/DesignTime/InformationSchema.fs

```diff
@@ -462,18 +468,15 @@ let getDbSchemaLookups(connectionString) =
                       Name = string row.["col_name"]
                       DataType = { Name = udtName
                                    Schema = schema.Name
-                                   ClrType = clrType }
+                                   ClrType = clrType
+                                   IsArray = string row.["col_data_type"] = "ARRAY" }
```

Comment:
This won't cut it for materialized views, see https://github.com/demetrixbio/FSharp.Data.Npgsql/pull/58/files#diff-e8040da32711ba0c71855707d78ec108R452-R457

Reply:
Thanks for feedback! In fact IsArray was cut, as ClrType will already contain information that is derived from this column.